Src/Support/Google.Apis.Auth.Tests/Google.Apis.Auth.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <!-- net452 is the earliest desktop version supported by xunit runner -->
+    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
```

Comment:
I intend to add a few more testing targets here later, including netcoreapp1.0. TODO added.